tests/test_lab_process_list.py

```diff
@@ -71,6 +82,21 @@ def test_process_list(cli_runner: CliRunner):
         expected_output
     ), result.output
 
+    # test with --state
+    result = cli_runner.invoke(
+        lab.ilab,
+        [
+            "--config=DEFAULT",
+            "process",
+            "list",
+            "--state",
+            "running",
+        ],
+    )
+    assert result.exit_code == 0, result.output
+    assert "63866b91-799a-42a7-af02-d0b68fddf19d" in result.output
+    assert "63866b92-799a-42a7-af02-d0b68fddf19d" not in result.output
+
```

Comment:
please add a test case that would confirm that Done processes are not removed from the list after first listing.